cpp/tests/test_odesecirvvs.cpp

```diff
@@ -759,6 +759,50 @@ TEST(TestOdeSECIRVVS, export_time_series_init)
                 MatrixNear(print_wrap(expected_results[0].get_groups().matrix()), 1e-5, 1e-5));
 }
 
+TEST(TestOdeSECIRVVS, export_time_series_init_old_date)
+{
+    TempFileRegister temp_file_register;
+    auto tmp_results_dir = temp_file_register.get_unique_path();
+    ASSERT_THAT(mio::create_directory(tmp_results_dir), IsSuccess());
+
+    auto num_age_groups = 6; // Data to be read requires RKI confirmed cases data age groups
+    auto model          = make_model(num_age_groups);
+
+    // set vaccinations to zero
+    model.parameters.get<mio::osecirvvs::DailyFirstVaccination<double>>().array().setConstant(0);
+    model.parameters.get<mio::osecirvvs::DailyFullVaccination<double>>().array().setConstant(0);
+    // set all compartments to zero
+    model.populations.array().setConstant(0.0);
```

Comment:
Is this necessary, or is population in export_... overridden?

Reply:
Yes its necessary.  Because of the old date, not all values are setted and have the old value initialized in `make_model`